x-pack/plugins/observability_solution/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -115,12 +115,15 @@ export function TimeseriesChart({
       }
     : undefined;
 
-  const endZoneColor = theme.darkMode ? theme.eui.euiColorLightShade : theme.eui.euiColorDarkShade;
+  const endZoneColor =
+    theme.colorMode === COLOR_MODES_STANDARD.light
```

Comment:
This should not be needed, I'd think. The tokens update based on color mode automatically.

Reply:
@mgadewoll, in that case, what would the _dynamic_ shade color be that I should use for this use case?

Reply:
@fkanout  Wait I'm also realizing that the condition used to be

```
theme.darkMode ? theme.eui.euiColorLightShade : theme.eui.euiColorDarkShade
```

and now it's

```
theme.colorMode === COLOR_MODES_STANDARD.light ? theme.euiTheme.colors.lightShade : theme.euiTheme.colors.darkShade;
```

Is that an expected change?

Reply:
Good catch @mgadewoll! I was logical about the condition when I rewrote it, but it's inverted. 
So, to keep the original intention, it should be 

`theme.colorMode === COLOR_MODES_STANDARD.dark ? theme.euiTheme.colors.lightShade : theme.euiTheme.colors.darkShade;
`

Reply:
🤔 Giving this some more thought, I'm wondering now if the colors are selected on purpose to ensure certain contrasts. If so we couldn't update to one token I'm afraid.

What exactly is this used for, you mind sharing a screenshot?

| colorMode | Token | Amsterdam | Borealis |
|------------|-------|-------------|-------------|
| DARK | lightShade | `#343741` | `#243147` |
| LIGHT | darkShade | `#69707D` | `#5A6D8C` | 

_this probably won't work_
| colorMode | Token | Amsterdam | Borealis |
|------------|-------|-------------|-------------|
| DARK | darkShade | `#98A2B3` | `#7186A8` |
| LIGHT | darkShade | `#69707D` | `#5A6D8C` |

Reply:
@fkanout I think let's keep it as is for now, no need to block anything. We'll have another look once we resolve the legacy shade colors. Sorry for the noise! 😅